tests/test_api.py

```diff
@@ -526,12 +527,47 @@ def test_api_component_400(client, api_path):
 
 
 def test_product_components_ofuri(client, api_path):
-    openssl = ComponentFactory(name="openssl")
-    curl = ComponentFactory(name="curl")
+    """test 'latest' filter on components"""
```

Comment:
So we're testing that this returns a certain number of components, can we also verify that it returns the right one please?